### PR TITLE
Fix a few typos in docstrings

### PR DESCRIPTION
**Pull request by pitmonticone**
## Changes proposed in this Pull Request

I've just fixed a few typo in docstrings and comments. 

## Checklist

- [x] I consent to the release of this PR's code under the GPLv3 license and non-code contributions under CC0-1.0 and CC-BY-4.0.
- [x] I tested my contribution locally and it seems to work fine.
- [x] Code and workflow changes are sufficiently documented.
- [x] Newly introduced dependencies are added to `envs/environment.yaml` and `envs/environment.docs.yaml`.
- [x] Changes in configuration options are added in all of `config.default.yaml` and `config.tutorial.yaml`.
- [x] Add a test config or line additions to `test/` (note tests are changing the config.tutorial.yaml)
- [x] Changes in configuration options are also documented in `doc/configtables/*.csv` and line references are adjusted in `doc/configuration.rst` and `doc/tutorial.rst`.
- [x] A note for the release notes `doc/release_notes.rst` is amended in the format of previous release notes, including reference to the requested PR.

### CODE REVIEW

File: scripts/build_natura_raster.py [hunk 1]
```diff
@@ -116,7 +116,7 @@ def get_transform_and_shape(bounds, res, out_logging):
 
 def unify_protected_shape_areas(inputs, natura_crs, out_logging):
     """
-    Iterates thorugh all snakemake rule inputs and unifies shapefiles (.shp) only.
+    Iterates thorough all snakemake rule inputs and unifies shapefiles (.shp) only.
```
Comment (pz-max) [inline scripts/build_natura_raster.py]: should be `through`?

**Review by pz-max — CHANGES_REQUESTED** [review]
Thanks for the spell lint :) just one minor change is required. Could you fix this?

**Comment by pz-max** [comment]
@pitmonticone  Could you also please confirm here that this contribution can be relicensed to AGPL in case the community decides so?
There are some relicensing discussions ongoing: https://github.com/pypsa-meets-earth/pypsa-earth/issues/693

File: scripts/build_natura_raster.py (hunk 1 above)
Comment (pitmonticone) [reply scripts/build_natura_raster.py]: Yep, sorry. I'll fix it in a moment.

**Comment by pitmonticone** [comment]
> Thanks for the spell lint :) just one minor change is required. Could you fix this?

Ops, sorry. Done.

**Review by pz-max — APPROVED** [review]
Thanks

**Comment by pitmonticone** [comment]
> @pitmonticone  Could you also please confirm here that this contribution can be relicensed to AGPL in case the community decides so?
> 
> There are some relicensing discussions ongoing: https://github.com/pypsa-meets-earth/pypsa-earth/issues/693

Confirmed in the discussion thread with 🏆. Thanks.

**Comment by pz-max** [comment]
Welcome as contributor @pitmonticone :1st_place_medal:

**Comment by pitmonticone** [comment]
Thank you very much @pz-max for the development and maintenance of all these great open-source research-quality packages!

Let me know if you believe it could be helpful if I join this GitHub organisation as a member. Happy to help when I find the time.

**Comment by pz-max** [comment]
@pitmonticone, thanks for the offer. It probably just makes sense to make you a org. member in case you prove to be a long-term contributor. But sure, we will inform you if it makes sense otherwise :+1:

**Comment by pitmonticone** [comment]
It sounds right to me. Thanks a lot for the quick review and merge.

**Pull request merged**